### PR TITLE
[watermark] Fix error: BUFFER_POOL_WATERMARK isn't enabled by default

### PR DESCRIPTION
**Pull request by stephenxs**
**- Why I did it**
Fix error: watermarkstat -t buffer_pool doesn't work

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-orchagent/enable_counters.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def enable_counters():
     enable_counter_group(db, 'PFCWD')
     enable_counter_group(db, 'PG_WATERMARK')
     enable_counter_group(db, 'QUEUE_WATERMARK')
+    enable_counter_group(db, 'BUFFER_POOL')
```
Comment (mykolaf) [inline dockers/docker-orchagent/enable_counters.py]: Correct counter group name is BUFFER_POOL_WATERMARK

```suggestion
    enable_counter_group(db, 'BUFFER_POOL_WATERMARK')
```

**Comment by mykolaf** [comment]
Need to change the title. The issue is not with `watermarkstat` and CLI, the cause is that the BUFFER_POOL_WATERMARK is not enabled by default.

**Comment by stephenxs** [comment]
> Need to change the title. The issue is not with `watermarkstat` and CLI, the cause is that the BUFFER_POOL_WATERMARK is not enabled by default.

Done.

**Review by mykolaf — APPROVED** [review]
LGTM

**Comment by stephenxs** [comment]
community PR [4882](https://github.com/Azure/sonic-buildimage/pull/4882) opened

**Pull request closed without merging**